backpack-compose/tokens.gradle.kts

```diff
@@ -142,4 +145,32 @@ tasks {
     this.group = group
     dependsOn(generateSizeTokens, generateColorTokens, generateTextTokens)
   }
+
+  val iconsClass = ClassName("net.skyscanner.backpack.compose.icons", "BpkIcons")
```

Comment:
It's appears for me to be more like resources, not tokens itself. Just like a components. It's also has icons package in material icons.

It may play its role if we move to tools like Svg2Compose

Reply:
that's fair - can we rename the class to `BpkIcon`?